CHANGELOG.rst

```diff
@@ -2,13 +2,23 @@
 Changelog
 =========
 
-Next release
+0.5.5 (2020-02-14)
 =======
 
 Changes
 -------
 
 * Update README.rst to make the example work
+* Bump Dockerfile to use Alpine 3.10 as base
+* Fix up ``load_source()`` call which doesn't exist in Python 3.5
+* Update versions of Python tested
+* Repair tests
+* pin to attrs 19.2 and fix deprecated arguments
+
+
+We are happy to add Matt Richter as maintainer - thanks a lot for stepping up!
```

Comment:
The construct "happy to do something" is a typically used as a confirmation of one's ability and willingness to do something; something like "We are very delighted to welcome Matt Richter on board as maintainer" would be more appropriate in this context.